tap_closeio/streams.py

```diff
@@ -165,13 +165,36 @@ def sync_activities(ctx):
     # calls that are in progress _while_ an extraction is happening, no
     # matter the replication frequency or call duration.
     offset_secs = ctx.config.get("activities_window_seconds", (60 * 60 * 24))
+
+    try:
+        # get date window from config
+        date_window = int(ctx.config.get("date_window", 15))
+        # if date_window is 0, '0' or None, then set default window size of 15 days
+        if not date_window:
```

Comment:
Put a warning logger before it to inform the same to the user.

Reply:
Added warning for invalid date window